.travis.yml

```diff
@@ -33,3 +36,13 @@ before_install:
 
 install:
         - true # HACK: fixes travis-CI lack of support for vendor/ + godeps
+
+after_success:
+  - set -e
+  - if [[ "${TRAVIS_BRANCH}" == "master" ]]; then
```

Comment:
a minor detail: you may also want to check for `"${TRAVIS_PULL_REQUEST}" == false`; otherwise travis will publish a docker image not only for commits to the master branch, but also for open PRs. which might not be exactly what we want.